scripts/t212_fetch.py

```
@@ -0,0 +1,326 @@
+    save_state(prefix, {"last_fetch": now.isoformat()})
+
+
+def main():
+    accounts = load_accounts()
+    print(f"Found {len(accounts)} configured account(s): {[a['prefix'].upper() for a in accounts]}")
+
+    for account in accounts:
+        fetch_account(account)
+
+    print("\n✅ All accounts synched. Launching run-all.sh for conversion...")
+    os.system("bash run-all.sh")
```

Ruff (0.15.9)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)